Tasks/DeployTestAgent/task.loc.json

```diff
@@ -111,12 +111,21 @@
       "helpMarkDown": "Optionally supply the path to vstf_testagent.exe from network or local location. If no path is provided, it will be downloaded from http://go.microsoft.com/fwlink/?LinkId=532768"
     },
     {
-      "name": "updateTestAgent",
+        "name": "updateTestAgent",
+        "type": "boolean",
+        "label": "ms-resource:loc.input.label.updateTestAgent",
+        "defaultValue": "true",
+        "required": false,
+        "helpMarkDown": "Optionally specify if test agent needs to be updated.",
+        "groupName": "advanced"
+    },
+    {
+      "name": "isDataCollectionOnly",
       "type": "boolean",
-      "label": "ms-resource:loc.input.label.updateTestAgent",
-      "defaultValue": "true",
+      "label": "ms-resource:loc.input.label.isDataCollectionOnly",
+      "defaultValue": "false",
       "required": false,
-      "helpMarkDown": "Optionally specify if test agent needs to be updated.",
+      "helpMarkDown": "Denotes if test agent needs to be used only for datacollection and not for running tests. Typically done on Application under test environments.",
```

Comment:
this does not match the non localized file ... this file must be auto generated by gulp instead of manual edits